test/malli/generator_test.cljc

```diff
@@ -314,10 +314,16 @@
       [:set {:min 10, :gen/max 20} int?]
       [:string {:gen/min 10, :max 20}]
 
+      [:+ {:min 10 :max 20} :int]
+      [:* {:min 10 :max 20} :int]
```

Comment:
I'd like to see tests for gen/min and gen/max as well.

Reply:
Good idea, adding those